src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/product_list.html.twig

```diff
@@ -0,0 +1,20 @@
+{% set isColumnLocationDisplayed = false %}
+{% set isColumnRefundedDisplayed = false %}
+{% for product in orderForViewing.products.products|slice(0, paginationNum) %}
+    {% if product.location is not empty %}
+        {% set isColumnLocationDisplayed = true %}
+    {% endif %}
+    {% if product.quantityRefunded > 0 %}
+        {% set isColumnRefundedDisplayed = true %}
+    {% endif %}
+{% endfor %}
```

Comment:
I think you can remove [those lines](https://github.com/PrestaShop/PrestaShop/blob/1.7.7.x/src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig#L26-L35) then

Reply:
They are set in products.html.twig so no need to re-set them.

I did the same in the controller to pass the variables to the template for the Ajax action